### DISCUSSION TITLE
Website feature and content priorities

### DISCUSSION THREAD

**Discussion by gusaus** [OpenProducer/angelcityjazz#54] [Planning & Strategy]
Let's use this issue to discuss content, feature, and maintenance priorities for 2021 (and beyond). Here's an outline based on what we've discussed. We can flesh it out and prioritize in the comments - 

#### ACJ Event Archives
Go [here](https://github.com/OpenProducer/angelcityjazz/discussions/54#discussioncomment-8424890) to evaluate and provide feedback
- [ ] https://github.com/OpenProducer/angelcityjazz/issues/61
- [ ] https://github.com/OpenProducer/angelcityjazz/issues/60
- [ ] https://github.com/OpenProducer/angelcityjazz/issues/56
- [ ] https://github.com/OpenProducer/angelcityjazz/issues/59

#### VR and video Archives
- [x] https://github.com/OpenProducer/angelcityjazz/issues/33
- [x] https://github.com/OpenProducer/angelcityjazz/issues/57
- [x] https://github.com/OpenProducer/angelcityjazz/issues/36

#### Layout and theme improvements
Go [here](https://github.com/OpenProducer/angelcityjazz/discussions/54#discussioncomment-6379476) to evaluate and provide feedback

#### Performance Enhancements and improvements
- [ ] https://github.com/OpenProducer/angelcityjazz/issues/46
- [ ] https://github.com/OpenProducer/angelcityjazz/issues/20
- [x] https://github.com/OpenProducer/angelcityjazz/issues/49
- [x] https://github.com/OpenProducer/angelcityjazz/issues/31

#### Hosting, maintenance, and support

- [x] https://github.com/OpenProducer/angelcityjazz/issues/41
- [x] https://github.com/OpenProducer/angelcityjazz/issues/35

**Comment by gusaus** [comment]
@angelcityjazz I've been going through and making several improvements and adjustments to the theme (on https://acjstg.wpengine.com/). I'll list them for you to review and check off in this discussion. 

**Single pages**
Set single pages to one column and increased the width from 780 to 1000px

- [ ] https://acjstg.wpengine.com/about/
- [ ] https://acjstg.wpengine.com/contact/
- [ ] https://acjstg.wpengine.com/donate/
- [ ] https://acjstg.wpengine.com/volunteer/
- [ ] https://acjstg.wpengine.com/cultural-equity-statement/
- [ ] https://acjstg.wpengine.com/privacy-policy/
  
**Woocommerce / account pages**
Set woocommerce account pages to one column and increased the width from 780 to 1000px

- [ ] https://acjstg.wpengine.com/my-account/
- [ ] https://acjstg.wpengine.com/cart/
- [ ] https://acjstg.wpengine.com/checkout/

**Woocommerce / shop pages**
Reduced the width from 1200 to 1000px

- [ ] https://acjstg.wpengine.com/product/unisex-organic-cotton-t-shirt/
- [ ] https://acjstg.wpengine.com/product/large-organic-tote-bag/

**Search**

- [ ] set one column page to 1000px wide
- [ ] changed settings so 'only' posts and current/upcoming 'events' show up in search results
- [ ] hid fields like author and category to make the results clean and consistent


Feel free to provide additional input and feedback in the comments (which are threaded). Once you sign off on everything, I'll push the files to live.

**Comment by gusaus** [comment]
As I was mentioning in https://github.com/OpenProducer/angelcityjazz/issues/56, my thought was to group together all archives (events, videos, audio, photos, press, news, posters/art, merch) and have a landing page (using a similar format). We could even customize the colors/banners to match whatever we were using that year. 

I took a little time to create pages for each year and played with the format a bit. Check the links and leave a comment (preferably in this comment thread) about the general ideal and other thoughts. 

- [ ] https://github.com/OpenProducer/angelcityjazz/issues/61
  - https://acjstg.wpengine.com/angel-city-jazz-fest/2020-acjf/ <-- per the issue, the event pages need to be reformatted
- [ ] https://github.com/OpenProducer/angelcityjazz/issues/60
  - https://acjstg.wpengine.com/angel-city-jazz-fest/2021-acjf/ 
- [ ] https://github.com/OpenProducer/angelcityjazz/issues/56
  - https://acjstg.wpengine.com/angel-city-jazz-fest/2022-acjf/ 
- [ ] https://github.com/OpenProducer/angelcityjazz/issues/59
   - https://acjstg.wpengine.com/angel-city-jazz-fest/2023-acjf/

**Reply by gusaus** [reply]
@angelcityjazz I had to rework the theme a bit for 'previous events' display, but here's what that looks like (without any customization) -
https://acjstg.wpengine.com/events/

A couple things to point out and evaluate - 

- the 2020 VR events have been archived since the format was modified to meet the needs of the virtual fest; we still have the videos archives on https://acjstg.wpengine.com/angel-city-jazz-fest/2020-acjf/ and https://acjstg.wpengine.com/watch/2020-vr-festival/
- the landing page doesn't really work or is needed (in my opinion) unless there are also upcoming events. Not sure if anything is planned (if not maybe we could curate/aggregate partner events?). 

Leave your thoughts as a comment whenever you can.

**Reply by gusaus** [reply]
> let's remove the past events listings on the events page

@angelcityjazz With regards to your email comment - do you mean hide the header? Actually think we should keep hiding the 'upcoming section/nav' until there are upcoming events (see the screenshot)

<img width="1379" alt="Events from October 29, 2023 – October 13, 2023 – Angel City Jazz 2024-02-16 15-53-11" src="https://github.com/OpenProducer/angelcityjazz/assets/113996/bca459ce-6dda-486b-8d90-05a7f3eb5112">

**Reply by gusaus** [reply]
> the 2020 archived festival is incorrect. It seems that the 2021 program is there instead. I think the 2020 fest should just link to the 202 VR fest page

Consider https://github.com/OpenProducer/angelcityjazz/discussions/54#discussioncomment-8424890 a rough draft. Wanted to get feedback on the general idea/layout before refining each page. 

Will do so now....

**Reply by gusaus** [reply]
@angelcityjazz What do you think about swapping out the banner with a cover image on the festival landing page? https://acjstg.wpengine.com/angel-city-jazz-fest/

We could swap back out with the 2024 banner (when ready) but I think an image might work in the meantime. Of course you'll probably be able to find a better image and might not like the duotone filter. 

Feel free to play around on the staging site. Let me know if you have any questions.

**Comment by angelcityjazz** [comment]
What I meant is to remove the past events from the event listings and
revert to the previous version of that page since the archived festival
schedules will be accessible on the festival page.
If people click on the festival page without selecting a specific year from
the submenu, are they going to directed to the latest/current schedule?
Also, once I add the 2024 schedule, will that add one more submenu item?

**Reply by gusaus** [reply]
What about leaving off the 'Events' menu item until we post some upcoming events (will there be any 'before' the 2024 fest)? So when we finalize and push the archive pages to the live site (https://angelcityjazz.com/) we'd only create the ACJF submenu items like we have on stage (https://acjstg.wpengine.com/). Does that make sense?

**Reply by gusaus** [reply]
> If people click on the festival page without selecting a specific year from the submenu, are they going to directed to the latest/current schedule?

Think it makes sense to keep displaying the latest fest via main menu item/primary landing page  by adding the schedule section, new art, playlist, etc. when ready.

Also makes sense to have the landing page display general info, programs, press, featured archives, etc. in between. That's what we're trying to do with https://acjstg.wpengine.com/angel-city-jazz-fest/ (<-- let me know what you think about splitting 'Watch' and 'Past Programs" into their own sections)

Our goal is to determine what we can/should implement in the short term (I'd like to like finalize and push the restructured landing page and archive pages by end of next week) and then figure what would be nice to roll in for the upcoming fest. 

Does that make sense and sound reasonable?

**Comment by angelcityjazz** [comment]
Yes. We have some events in May but Inwull only add them in april.

**Comment by gusaus** [comment]
@angelcityjazz I was going to circle back tomorrow about [finalizing the archives](https://github.com/OpenProducer/angelcityjazz/discussions/54#discussioncomment-8424890) (still will) but just saw the announcement about Youth Artist Completion coming back in the newsletter (congrats on the grant!). 

Also noticed you already updated https://angelcityjazz.com/young-artist-competition/ (and the menu) and wanted to make sure you're happy with what's there.

My first take after skimming over is we might want to make the text and background constant with the rest of the site. I can clone to dev or stage and play around.

**Comment by angelcityjazz** [comment]
How about just a collage banner without text (see current version). I would
need to resize the collage though. WHat size would work best?
Another thing about the archives: is there a way to consolidate the pre
2019 pdfs programs with the 2020 to 2023 archived schedules? We should at
least have all of them accessible formt he same place.

**Reply by gusaus** [reply]
> How about just a collage banner without text (see current version). 

I don't think that works since many of the images are small already (and won't look right on different devices). I'd go back to using an image that works well as a background (do you like the duotone filter on https://acjstg.wpengine.com/young-artist-competition/?)

> Is there a way to consolidate the pre 2019 pdfs programs with the 2020 to 2023 archived schedules? We should at
least have all of them accessible formt he same place.

So all the programs from 2019 and before are on https://acjstg.wpengine.com/angel-city-jazz-fest/ (how do they look in their own section?), but I'm not sure how to link to the pages that have archives (aside from having in the submenu). Since we don't have posters (right?), we couldn't add to the 'past programs' and have them link to archives. We could however create a new section (recent archives?) and have an image for each that would link to the archive. 


With regards the archive pages, I've been thinking its overkill to have the same about, press, and sponsors sections on the archives. So I removed those on https://acjstg.wpengine.com/angel-city-jazz-fest/2023-acjf/ but created a section with the spotify embed and a Tiled Gallery block (that actually has a similar look as the collage). I just selected some random images (just to evaluate the basic look and feel).

Let me know what you think about all of the above.
